guides/common/modules/proc_adding-project-server-to-awx-as-a-dynamic-inventory-item.adoc

```diff
@@ -13,6 +13,9 @@ For more information about managing users, roles, and permission filters, see {A
 .Procedure
 . In the {awx} web UI, create a credential for your {Project}.
 For more information about creating credentials, see http://docs.ansible.com/automation-controller/latest/html/userguide/credentials.html#add-a-new-credential[Add a New Credential] and http://docs.ansible.com/automation-controller/latest/html/userguide/credentials.html#red-hat-satellite-6[{ProjectName} Credentials] in the _Automation Controller User Guide_.
+ifdef::orcharhino[]
+Note that this part of the Ansible documentation also applies to {Project}.
+endif::[]
```

Comment:
ifdef here on purpose instead of ifndef?

Reply:
Yes, because the docs on ansible.com mention Red Hat Satellite.